downloads/pre-release/index.php

```diff
@@ -48,8 +48,7 @@
 <p>Ubuntu: Keyman for Linux can be installed via <a href='https://launchpad.net/~keymanapp/+archive/ubuntu/keyman-daily'>launchpad</a>:</p>
 <pre class='language-bash code'><code>
 sudo add-apt-repository ppa:keymanapp/keyman-daily
-sudo apt-get update
-sudo apt-get install keyman onboard-keyman</code></pre>
+sudo apt install keyman onboard-keyman</code></pre>
```

Comment:
How come the 14/index.php kept the `sudo apt upgrade` step?

Reply:
I'm a little bit torn if that's necessary or not. Usually it shouldn't be necessary...